as/itests/org.jboss.tools.as.ui.bot.itests/src/org/jboss/tools/as/ui/bot/itests/archives/VariousProjectsArchiving.java

```diff
@@ -47,6 +48,7 @@ public class VariousProjectsArchiving extends ArchivesTestBase {
 	
 	@BeforeClass
 	public static void setup() {
+		Preferences.set("org.eclipse.debug.ui", "Console.limitConsoleOutput", "false");
```

Comment:
Can it be moved to some parent class before so that all test classes can share it?

Reply:
done